code/modules/food_and_drink/snacks.dm

```diff
@@ -541,6 +541,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/soup)
 
 			src.bites_left--
 			if (!bites_left)
+				new src.dropped_item(src.loc)
```

Comment:
```suggestion
				new src.dropped_item(get_turf(src))
```
what if someone eats chips while holding both

Reply:
Yeah, that'd break it.